sklearn/tests/test_import_deprecations.py

```diff
@@ -13,6 +13,16 @@
 @pytest.mark.parametrize('deprecated_path, importee', (
     ('sklearn.neural_network.rbm', 'BernoulliRBM'),
     ('sklearn.neural_network.multilayer_perceptron', 'MLPClassifier'),
+
+    ('sklearn.cluster.affinity_propagation_', 'AffinityPropagation'),
+    ('sklearn.cluster.bicluster', 'SpectralBiclustering'),
+    ('sklearn.cluster.birch', 'Birch'),
+    ('sklearn.cluster.dbscan_', 'DBSCAN'),
+    ('sklearn.cluster.hierarchical', 'FeatureAgglomeration'),
+    ('sklearn.cluster.k_means_', 'KMeans'),
+    ('sklearn.cluster.mean_shift_', 'MeanShift'),
+    ('sklearn.cluster.optics_', 'OPTICS'),
+    ('sklearn.cluster.spectral', 'SpectralClustering'),
```

Comment:
Haven't checked if this list is complete.

Reply:
This seems to be leading to some confusion.

Can we simplify this check by only doing a module import:

```
import {deprecated_path}
```

So we do not need the `importee`?

Reply:
That would only check that 

```import sklearn.neural_network.rbm```

raises a warning, but not that we can still import the stuff we used to from `rbm`. I think it's important to still ensure that

Reply:
Then we need a comment somewhere saying "We only need to import one thing from each file"

Reply:
done